packages/expo-media-library/package.json

```diff
@@ -43,5 +43,15 @@
   "peerDependencies": {
     "expo": "*",
     "react-native": "*"
+  },
+  "codegenConfig": {
+    "name": "expo-media-library",
+    "type": "modules",
+    "jsSrcsDir": "./src",
```

Comment:
I'm wondering if `name` and `jsSrcsDir` are necessary. `expo-media-library`, like other Expo Modules, doesn't use codegen so there is no need for codegen to look for spec files 🤔 Can you try?

Reply:
Hey @tsapeta - removing any of these three fields causes `pod install` to fail, unfortunately. 

I agree it would be better if we could skip this, but I think we need to include them in order for React Native to pick up whatever it's doing with `modulesConformingToProtocol`

Reply:
Cool, thanks for checking this!